docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -45,52 +47,48 @@ Then complete the docker registryconnection details:
 3. Click the checkbox to desigate this registry as push registry
 4. Your dockerhub namespace (which is usually the same as your username)
 
-The connection will be tested to validate your credentials and are stored in a Kubernetes secret within the Codewind service.
+The connection is tested to validate your credentials, and these are then stored in a Kubernetes secret within the Codewind service.
 
 ## Create a new project
 
-Navigate to the Codewind panel and right click the deployment, then choose `create new project`:
+1\. Navigate to the Codewind panel, right-click the deployment, and then click `create new project`:
 
 ![Adding new remote project](./images/remoteeclipse/newProject.png)
 
-Select the template type, project name and your new project should be built and after a few seconds begin running in the cloud.
+2\. Select the template type, and name your project. Your new project is built and after a few seconds begins running in the cloud.
```

Comment:
Comma not needed after "template type." The phrase "and name your project" is not a standalone independent clause, so just the conjunction will suffice.